ARTCC/High/CDR1 N601(D).txt

```diff
@@ -0,0 +1,6 @@
+;N601 CDR FILLET
+CDR1 N601(D) FL165-195 N055.17.35.000 W002.54.27.000 N055.17.24.000 W002.45.32.000
```

Comment:
I propose "Conditional - N601 (D) FL165-FL195"

Reply:
Fixed in 3107948cb7110f73fbb92863c3598797527b8d43